entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityAttributeMapping.java

```diff
@@ -30,14 +34,22 @@
         config.getConfigList(ATTRIBUTE_MAP_CONFIG_PATH).stream()
             .collect(
                 toUnmodifiableMap(
-                    conf -> conf.getString("name"), conf -> conf.getString("subDocPath"))));
+                    conf -> conf.getString("name"), conf -> conf.getString(SUB_DOC_PATH))),
+        config.getConfigList(ATTRIBUTE_MAP_CONFIG_PATH).stream()
```

Comment:
Actually, given that this is a new feature and has no compatibility requirements, do we even need to add this to config at all? Why not always retrieve from AS (the subdoc path from config we needed to continue supporting for backwards compat)

Reply:
Agree, apart from backward compatibility i thought it is added for performance concerns also.  There is client side caching in client, but i thought through config we want to limit calls to attribute service.

Reply:
The cached calls shouldn't have any significant perf impacts - it would be 1 call per tenant per 15m, by default (and that cache is shared by the other part of the mapping on col name, anyway)